Misc/NEWS.d/next/Core and Builtins/2019-12-30-15-56-07.bpo-36051.imaVlq.rst

```diff
@@ -0,0 +1 @@
+Drop the GIL during large ``bytes.join`` operations.
```

Comment:
Add your name in ACKS (see https://devguide.python.org/pullrequest/#crediting),
and add "Patch by ...." here.

Reply:
I've already added myself to ACKS, but have now added "Patch by ...".

Reply:
I can not find "Bruce Merry" in ACKS.
https://github.com/python/cpython/blob/master/Misc/ACKS

Reply:
That's the master version of ACKS. I'm adding myself in this pull request: https://github.com/bmerry/cpython/blob/bpo-36051/Misc/ACKS#L1109